invenio_previewer/templates/invenio_previewer/zip.html

```diff
@@ -51,7 +51,7 @@
 <div class="panel-heading">
   <div class="panel-title">
     <i class="fa fa-file-zip-o"></i> {{ file.filename }}
-    <div id="fullScreenMode" title="{{ _('Full screen') }}" class="pull-right"><i class="fa fa-fw fa-arrows-alt"></i></div>
+    <div id="fullScreenMode" title="{{ _('Full screen') }}" class="hidden pull-right"><i class="fa fa-fw fa-arrows-alt"></i></div>
```

Comment:
@topless do you remember why this`hidden`?

Reply:
I found it redundant to display a full screen control for the contents of a zip file. We can always make it visible.